src/mumble/mumble_pch.hpp

```diff
@@ -81,6 +81,9 @@
 #define interface struct
 #endif
 
+#ifndef _WIN32_WINNT
```

Comment:
Perfect, it's working.

```
mumble_pch.hpp:85:0: warning: "_WIN32_WINNT" redefined
 # define _WIN32_WINNT 0x0600 // We need to do this for MinGW on Linux
 ^
In file included from /usr/lib/mxe/usr/i686-w64-mingw32.static/include/crtdefs.h:10:0,
                 from /usr/lib/mxe/usr/i686-w64-mingw32.static/include/stddef.h:7,
                 from /usr/lib/mxe/usr/lib/gcc/i686-w64-mingw32.static/5.4.0/include/stddef.h:1,
                 from /usr/lib/mxe/usr/lib/gcc/i686-w64-mingw32.static/5.4.0/include/c++/cstddef:45,
                 from /usr/lib/mxe/usr/i686-w64-mingw32.static/qt5/include/QtCore/qglobal.h:46,
                 from /usr/lib/mxe/usr/i686-w64-mingw32.static/qt5/include/QtCore/QtCore:4,
                 from mumble_pch.hpp:28:
/usr/lib/mxe/usr/i686-w64-mingw32.static/include/_mingw.h:225:0: note: this is the location of the previous definition
 #define _WIN32_WINNT 0x502
 ^
```

According to [MSDN](https://msdn.microsoft.com/it-it/library/windows/desktop/aa383745(v=vs.85).aspx), _0x502_ is Windows Server 2003 with SP1 or Windows XP with SP2...